assets/app/view/game/round/stock.rb

```diff
@@ -300,14 +300,14 @@ def render_sell_input(company)
             ))
             store(:selected_company, nil, skip: true)
           end
-          button_text = if @step.respond_to?(:button_text)
-                          @step.button_text
-                        else
-                          "Sell #{@selected_company.sym} to Bank for #{@game.format_currency(price)}"
-                        end
+          sell_share_text = if @step.respond_to?(:button_text)
```

Comment:
I meant the method "button_text" was too generic and should be called "sell_share_text" or something like that. Sorry for the confusion.

Reply:
not needed now that it's a chooce